{
  "url": "https://api.github.com/repos/facebook/react/issues/935",
  "repository_url": "https://api.github.com/repos/facebook/react",
  "labels_url": "https://api.github.com/repos/facebook/react/issues/935/labels{/name}",
  "comments_url": "https://api.github.com/repos/facebook/react/issues/935/comments",
  "events_url": "https://api.github.com/repos/facebook/react/issues/935/events",
  "html_url": "https://github.com/facebook/react/issues/935",
  "id": 25860608,
  "node_id": "MDU6SXNzdWUyNTg2MDYwOA==",
  "number": 935,
  "title": "Better error message when returning two or more elements",
  "user": {
    "login": "jkleiser",
    "id": 107986,
    "node_id": "MDQ6VXNlcjEwNzk4Ng==",
    "avatar_url": "https://avatars2.githubusercontent.com/u/107986?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/jkleiser",
    "html_url": "https://github.com/jkleiser",
    "followers_url": "https://api.github.com/users/jkleiser/followers",
    "following_url": "https://api.github.com/users/jkleiser/following{/other_user}",
    "gists_url": "https://api.github.com/users/jkleiser/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/jkleiser/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/jkleiser/subscriptions",
    "organizations_url": "https://api.github.com/users/jkleiser/orgs",
    "repos_url": "https://api.github.com/users/jkleiser/repos",
    "events_url": "https://api.github.com/users/jkleiser/events{/privacy}",
    "received_events_url": "https://api.github.com/users/jkleiser/received_events",
    "type": "User",
    "site_admin": false
  },
  "labels": [

  ],
  "state": "closed",
  "locked": false,
  "assignee": null,
  "assignees": [

  ],
  "milestone": null,
  "comments": 4,
  "created_at": "2014-01-18T13:47:27Z",
  "updated_at": "2014-05-23T09:04:00Z",
  "closed_at": "2014-05-23T09:04:00Z",
  "author_association": "NONE",
  "body": "Since I'm quite new to React, I tried for my render function to return something like this:\n\n```\n<div>{this.state.s1}</div><div className=\"y\">{this.state.s2}</div>\n```\n\nIn Safari I got the message \"Error: Parse Error: Line 9: Unexpected identifier\". It took quite a while before I found out that it was the second 'div' element that was the problem. If possible, the error message should have mentioned the second element. \"Unexpected identifier\" is just confusing.\n",
  "closed_by": {
    "login": "sophiebits",
    "id": 6820,
    "node_id": "MDQ6VXNlcjY4MjA=",
    "avatar_url": "https://avatars2.githubusercontent.com/u/6820?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/sophiebits",
    "html_url": "https://github.com/sophiebits",
    "followers_url": "https://api.github.com/users/sophiebits/followers",
    "following_url": "https://api.github.com/users/sophiebits/following{/other_user}",
    "gists_url": "https://api.github.com/users/sophiebits/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/sophiebits/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/sophiebits/subscriptions",
    "organizations_url": "https://api.github.com/users/sophiebits/orgs",
    "repos_url": "https://api.github.com/users/sophiebits/repos",
    "events_url": "https://api.github.com/users/sophiebits/events{/privacy}",
    "received_events_url": "https://api.github.com/users/sophiebits/received_events",
    "type": "User",
    "site_admin": false
  }
}
